chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+it is broadly agreed that this is, on average, the most time-consuming step in a project's data work \citep{dasu2003exploratory}.
+This chapter describes the different tasks involved in the cleaning process,
+and outlines the workflow our team developed to make the process efficient and transparent.
+
+% What is data cleaning
+This chapter describes all tasks needed to make newly acquired data ready for analysis.
+The first section will guide on making your data \textit{tidy},
+so the relationship between rows and columns in a data set is well-defined.
+This includes removing duplicated observations.
+The second section describes quality assurance checks, which are necessary to verify the accuracy of your data
+After that, identifying information is typically no longer needed,
+the third section outlines steps to remove direct identifiers.
+Finally, the last section discusses how to explore each variable in your dataset and 
+make sure that the dataset is as well documented and easy to use as possible.
+
+
+\end{fullwidth}
+
+%------------------------------------------------
+
+\subsection{Before you get started}
+
+% Don't mess with the raw data
+The starting point for all tasks described in this chapter is the raw data,
+containing the exact data files acquired, with no changes or additions.
+Be mindful of how and where these files are stored
+as they cannot be re-created and nearly always contain confidential information.
+Raw data files must \textit{never} be edited directly.
+If the raw data contains confidential information,
+any tasks that precede de-identification must be carried on encrypted data.
+ 
+% Why cleaning is necessary
+Raw data files may be received in a variety of formats, 
+and are often not easy to handle using statistical software.
+The data cleaning tasks described in this chapter will prepare the raw data
+for quantitative analysis.
+To make this complex process less error-prone,
+we recommend that no changes be made to the content of data at this point
+(with one exception that will be carefully discussed).
+Correcting outliers, imputing values and creating derived indicators
+are all research decisions, and are therefore considered part of analysis (discussed in the next chapter).
+Therefore, the main output at this stage, the cleaned dataset,
+contains almost exactly same information as the raw data,
+but in a format that is easier to interact with.  
+
+\section{Identifying the identifier(s)}
+
+% Intro
+The very first step in creating statistical software-friendly data sets
+is understanding the data acquired,
+and translating this understanding into an intuitive format.
+This section discusses what steps may be needed to make sure that each row
+in a dataset represents one observation.
+This may seem trivial, but should not be taken for granted.
+Getting to such a format may be harder than expected,
+and the \textbf{unit of observation}\sidenote{\textbf{Unit of observation:}
+	the unit described by the data. \url{
```

Comment:
```suggestion
	the unit described by the data, often the same as the unit of each row in the dataset. \url{
```
We cannot say only "_unit of each row in the dataset_" as that is not the definition, but my experience from training is that it is adding "_unit of each row_" that makes people not already aware of this concept get what it means.

Reply:
Maybe "ideally" instead of "often"?

Reply:
Ideally when you use datasets, yes. But unit of observation also exits in non-relational datasets. For example, a unit of observation can be a satellite image, it can be a document, in qualitative research it is the respondent itself, etc. 

So in all datasets the row is ideally the unit of observations, but not all unit of observations are ideally a row.

What about:
```suggestion
	the unit described by the data (in datasets, it is ideally what each row represents). \url{
```

Reply:
done in d2cd334